docs/howto/error-reporting.txt

```diff
@@ -138,12 +138,25 @@ exception raised, each `traceback frame`_’s local variables, and the
 
 However, sometimes certain types of information may be too sensitive and thus
 may not be appropriate to be kept track of, for example a user's password or
-credit card number. So in addition to filtering out settings that appear to be
-sensitive as described in the :setting:`DEBUG` documentation, Django offers a
-set of function decorators to help you control which information should be
-filtered out of error reports in a production environment (that is, where
-:setting:`DEBUG` is set to ``False``): :func:`sensitive_variables` and
-:func:`sensitive_post_parameters`.
+an API token. Django cleanses sensitive data from settings and
+:attr:`.HttpRequest.META` as described in the :setting:`DEBUG` documentation.
+
+If there is other sensitive information you would like to cleanse, such as a
+credit card number, Django offers a set of function decorators to help you
+control information which should be filtered out of error reports in a
+production environment (that is, where :setting:`DEBUG` is set to ``False``):
+:func:`sensitive_variables` and :func:`sensitive_post_parameters`.
+
+.. versionchanged:: 3.0
+
+    Django cleanses settings and :attr:.HttpRequest.META that appear sensitive,
```

Comment:
I didn't mean in my previous comments to duplicate the exisiting text, rather to rephrase the section.